### PR TITLE
added custom HTTP-Methods for create and update

### PR DESCRIPTION
**Pull request by matsluni**
Hi,

for a current task I need the possibility to set a custom HTTP method for a create operation, in my case 'put' instead of 'post'. I saw a similar request in #362.  

I gave it a try in this pull request.  The change I did works for me, so please consider merging if you think this is a useful addition.

If you think another approach for the same goal is better just get back to me.
I also tried to add tests for this in CreateQueriesSpec.js and UpdateQueriesSpec.js but couldn't figure out to set the xxxMethod on the entity. So, if you could give me a hint I will add this as well.

Kind regards,
Matthias

### CODE REVIEW

File: src/javascripts/ng-admin/Crud/repository/CreateQueries.js [hunk 1]
```diff
@@ -24,12 +24,14 @@ define(function (require) {
...
     CreateQueries.prototype.createOne = function (view, rawEntity) {
-        return this.Restangular
-            .oneUrl(view.entity.name(), this.config.getRouteFor(view.entity, view.getUrl(), view.type))
-            .customPOST(rawEntity)
-            .then(function (response) {
-                return response.data;
-            });
+
+        var method = view.entity.createMethod(),
+            url = this.Restangular.oneUrl(view.entity.name(), this.config.getRouteFor(view.entity, view.getUrl(), view.type)),
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/repository/CreateQueries.js]: The variable name isn't well chosen. Use `resource` instead. Same in `UpdateQueries`

**Comment by matsluni** [comment]
Ok, I changed the variable name to resource. Is there anything more I can do?

**Comment by manuquentin** [comment]
Can you rebase please ?

**Comment by matsluni** [comment]
I tried to rebase and saw that you moved some pieces around! ;-) I have look closer to rebase correctly.

**Comment by jeromemacias** [comment]
You will need to do all this modifications on the new https://github.com/marmelab/admin-config repository. Sorry for inconvenience.

**Pull request closed without merging**